Test/Test/Data/ExceptionHandler.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace CSharpTest
+{
+    public class ExceptionHandler : IExceptionHandler
+    {
+        public void ExitProgram()
+        {
+            Console.WriteLine("Incorrect data - please, check data file for errors and try again");
+            System.Environment.Exit(1);
```

Comment:
this will cause program to stop - the user won't get a chance to read the comment